test/e2e/puppeteer.js

```diff
@@ -6,8 +6,6 @@ const jimp = require( 'jimp' );
 const fs = require( 'fs' );
 
 const port = 1234;
-const pixelThreshold = 0.1; // threshold error in one pixel
-const maxFailedPixels = 0.05; // total failed pixels
 
 const networkTimeout = 600;
```

Comment:
You probably need to increase `renderTimeout` on MacOS because seems that this machines have a slower CPU in Github Actions.